### PR TITLE
[argtable] add recipe

### PR DESCRIPTION
**Pull request by M-PERSIC**
[libargtable2](https://sourceforge.net/projects/argtable/) v2.13 and [libargtable3](https://github.com/argtable/argtable3) v3.2.2.

### CODE REVIEW

File: A/argtable/argtable@2.13/build_tarballs.jl [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+
+include("../common.jl")
+
+version = v"2.13"
```
Comment (giordano) [inline A/argtable/argtable@2.13/build_tarballs.jl:6]:
```suggestion
version_string = "2.13"
version = VersionNumber(version_string)
```

File: A/argtable/argtable@2.13/build_tarballs.jl [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource(
+        "https://downloads.sourceforge.net/project/argtable/argtable/argtable-2.13/argtable2-13.tar.gz",
```
Comment (giordano) [inline A/argtable/argtable@2.13/build_tarballs.jl:11]:
```suggestion
        "https://downloads.sourceforge.net/project/argtable/argtable/argtable-$(version_string)/argtable$(version.major)-$(version.minor).tar.gz",
```

File: A/argtable/argtable@2.13/build_tarballs.jl [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+        "8f77e8a7ced5301af6e22f47302fdbc3b1ff41f2b83c43c77ae5ca041771ddbf",
+    ),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/argtable2-13
```
Comment (giordano) [inline A/argtable/argtable@2.13/build_tarballs.jl:18]:
```suggestion
cd $WORKSPACE/srcdir/argtable*
```

File: A/argtable/argtable@2.13/build_tarballs.jl [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+install_license COPYING
+CC=gcc
+CXX=g++
```
Comment (giordano) [inline A/argtable/argtable@2.13/build_tarballs.jl:21]: Why?
```suggestion
```

File: A/argtable/argtable@3.2.2/build_tarballs.jl [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+        "55c4f6285e2f9b2f0cc96ae8212b7b943547c3dd",
+    ),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/argtable3
+install_license LICENSE
+CC=gcc
+CXX=g++
```
Comment (giordano) [inline A/argtable/argtable@3.2.2/build_tarballs.jl:21]: Why?
```suggestion
```
This is also [the wrong way](https://docs.binarybuilder.org/stable/build_tips/#Using-GCC-on-macOS-and-FreeBSD) to set the compilers when using CMake

Comment (M-PERSIC) [reply A/argtable/argtable@3.2.2/build_tarballs.jl:21]: Ah, apologies

**Pull request merged**